examples/sdk/reactNative/ios/rnbtexample/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
if we're adding it - we need to extend the list - example: https://github.com/backtrace-labs/backtrace-cocoa/blob/master/Sources/Resources/PrivacyInfo.xcprivacy